BraveRewardsUI/Wallet/WalletViewController.swift

```diff
@@ -189,11 +203,19 @@ class WalletViewController: UIViewController, RewardsSummaryProtocol {
         }
       }
       
+      self.state.ledger.listRecurringTips { [weak self] in
+        guard let self = self else { return }
+        guard let recurringTip = $0.filter({ $0.id == host && $0.rewardsCategory == .recurringTip }).first else { return }
+        
+        self.publisherSummaryView.monthlyTipView.batValueView.amountLabel.text = "\((recurringTip.contributions.first?.value ?? 0))"
```

Comment:
I'd say we should probably guard against showing this if we dont have an amount. So should unwrap recurringTip.contributions.first?.value before unhiding